app/services/access_token_verifier.rb

```diff
@@ -28,6 +28,11 @@ def identity
 
   attr_reader :http_authorization_header
 
+  def valid?
```

Comment:
In other usages `valid?` is dynamic and re-runs validations, and it's probably preferable to keep that consistent. Could we drop the repeated `valid?` instead?

Reply:
Yeah, I assume you're suggesting something like the first "Alternative solution" in the original pull request comment? I'm open to that as well.

Reply:
Although, the more I think about it, the more I like the second alternative option I proposed, since that helps provide stronger guarantees and avoids _assuming_ that `identity` would only be called after a submission validation.

Reply:
yeah, I think I agree there

Reply:
Updated in 93edf8f.